src/commands/cloud-run/README.md

```diff
@@ -0,0 +1,38 @@
+## Troubleshooting Cloud Run Instrumentation
```

Comment:
Missing `Configuration` section for credentials, env variables for site, api key, etc.